index.src.html

```diff
@@ -396,19 +396,25 @@
   <pre class="idl">
     [Exposed=Window, SecureContext]
     dictionary CredentialRequestOptions {
-      boolean unmediated = false;
+      CredentialMediationRequirement mediation = "allowed";
     };
   </pre>
-  <dl dfn-for="CredentialRequestOptions">
-    :   <dfn dict-member>unmediated</dfn>
-    ::  If `true`, the user agent will only attempt to provide a {{Credential}} without user
-        interaction. It MUST NOT present the user with any visible prompt to grant access to a
-        {{Credential}}: if the user has opted-into always giving a particular site access to a
-        particular set of credentials, they will be provided. If not, the promise will resolve
-        with `null`. For processing details, see the algorithm defined in
-        [[#algorithm-request]].
+  <dl dfn-for="CredentialRequestOptions" dfn-type="dict-member">
+    :   <dfn>mediation</dfn>
+    ::  This property specifies the mediation requirements for a given credential request. The
+        meaning of each enum value are described below in {{CredentialMediationRequirement}}.
```

Comment:
nit: "meaning of each enum value **is**" or "**meanings** of **the enum values** are"